content/src/logic/database-queries/snapshot-cleaner-queries.ts

```diff
@@ -0,0 +1,18 @@
+import SQL from 'sql-template-strings'
+import { AppComponents } from '../../types'
+
+export async function filterContentFileHashes(
```

Comment:
the name of this function may be ambiguous, is it keeping or filtering out the content file's hashes?